src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
```

Comment:
Using a LinkedHashMap is not recommended according to the checklist. Try to solve the task without using Map or any other similar data structures.